pkg/review/commit_review_status.go

```diff
@@ -22,8 +22,13 @@ import (
 	"fmt"
 
 	"github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
 )
 
+// githubPrApproved is the value GitHub set the `reviewDecision` field to
+// when a Pull Request has been approved by a reviewer.
+const githubPrApproved = "APPROVED"
```

Comment:
The conventional Go capitalization would be "githubPRApproved", per https://google.github.io/styleguide/go/decisions#initialisms

Reply:
done.